distro/pkg/deb/rules

```diff
@@ -13,6 +13,7 @@ NPROCS:=$(shell grep -c ^processor /proc/cpuinfo)
 override_dh_auto_configure:
 	dh_auto_configure -- \
 		-DCMAKE_BUILD_TYPE:String="RelWithDebInfo" \
+		-DCMAKE_BUILD_WITH_INSTALL_RPATH=ON \
```

Comment:
Could you please add some details to justify this change? Having to customize `LD_LIBRARY_PATH` for the tests is not ideal.

Reply:
We want to make the builds reproducible. 
For reference:
https://wiki.debian.org/ReproducibleBuilds/Howto

https://reproducible-builds.org/docs/deterministic-build-systems/

> **CMake notes**
The default configuration of CMake makes the build directory part of the build environment. Here are some known issues and recommendations:
CMake sets a RPATH for binaries that link to a library in the the same project. Even when this is stripped at installation time, the build-id section will be different. Possible workarounds:
Users can set [CMAKE_SKIP_RPATH=ON](https://cmake.org/cmake/help/latest/variable/CMAKE_SKIP_RPATH.html) to disable the use of RPATH. Disadvantage: programs from the build directory cannot be run without setting LD_LIBRARY_PATH.
Projects can set [CMAKE_BUILD_WITH_INSTALL_RPATH=ON](https://cmake.org/cmake/help/latest/variable/CMAKE_BUILD_WITH_INSTALL_RPATH.html) to ensure a deterministic RPATH. Disadvantage: programs from the build directory cannot be run without setting LD_LIBRARY_PATH.

Reply:
Fine, if you think there is any advantage to this. But it should be possible to set that `LD_LIBRARY_PATH` in this script, no? My guess would be that you can just set it in `override_dh_auto_test` when running the tests. But if not, there must be some other suitable package build step.